File: android/app/src/main/java/com/masreferenceapp/network/NetworkLocalNetwork.java
package com.masreferenceapp.network;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.ReactMethod;
import com.masreferenceapp.Status;


public class NetworkLocalNetwork extends ReactContextBaseJavaModule {
    ReactApplicationContext context;

    public NetworkLocalNetwork(ReactApplicationContext context) {
        super(context);
        this.context = context;
    }

    @NonNull
    @Override
    public String getName() {
        return "NetworkLocalNetwork";
    }

    @ReactMethod(isBlockingSynchronousMethod = true)
    public String putString(){
        return Status.status("OK", "Message");
    }
}

File: android/app/src/main/java/com/masreferenceapp/network/NetworkLocalNetwork.kt
package com.masreferenceapp.network

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule
import com.facebook.react.bridge.ReactMethod
import com.masreferenceapp.ReturnStatus
import com.masreferenceapp.Status

class NetworkLocalNetwork(var context: ReactApplicationContext) : ReactContextBaseJavaModule(
    context
) {
    override fun getName(): String {
        return "NetworkLocalNetwork"
    }

    @ReactMethod(isBlockingSynchronousMethod = true)
    fun putString(): String {
        return Status.status("OK", "Message")
    }

        
    @ReactMethod(isBlockingSynchronousMethod = true)
    fun access(): String {
        val r = ReturnStatus("OK", "Android code stub.")
        return r.toJsonString()
    }

        
    @ReactMethod(isBlockingSynchronousMethod = true)
    fun hostHttpServer(): String {
        val r = ReturnStatus("OK", "Android code stub.")
        return r.toJsonString()
    }

        
    @ReactMethod(isBlockingSynchronousMethod = true)
    fun hostTcpServer(): String {
        val r = ReturnStatus("OK", "Android code stub.")
        return r.toJsonString()
    }

        
    @ReactMethod(isBlockingSynchronousMethod = true)
    fun hostUDPServer(): String {
        val r = ReturnStatus("OK", "Android code stub.")
        return r.toJsonString()
    }

    //@method

}